src/main/kotlin/org/axonframework/intellij/ide/plugin/actions/SimulateErrorAction.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ *  Copyright (c) 2022. Axon Framework
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.axonframework.intellij.ide.plugin.actions
+
+import com.intellij.openapi.actionSystem.AnAction
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.application.ReadAction
+import org.axonframework.intellij.ide.plugin.AxonIcons
+
+/**
+ * Action that can be invoked using the "Simulate exception" action.
```

Comment:
If this is used for simulating/testing, why is it part of the `src` directory?

Reply:
Sharp, I'm removing it. Testing the window will become a bit harder though